dev/ProjectReunion_DLL/dllmain.cpp

```diff
@@ -8,8 +8,22 @@
 
 #include <../Detours/detours.h>
 
+static bool IsPackagedProcess()
```

Comment:
Isn't this like the 4th or 5th copy of this we've made?

Reply:
Lol 2nd or maybe 3rd. I'm planning to centralize this (and a few other things) but waiting on a couple of naming decisions first.

There's a laundry list of refactoring I've got my eye on. Planning to put some time in after I finish some critical path work.